php/src/Communicator.cpp

```diff
@@ -2,12 +2,13 @@
 // Copyright (c) ZeroC, Inc. All rights reserved.
 //
 
-#include <Communicator.h>
-#include <Logger.h>
-#include <Properties.h>
-#include <Proxy.h>
-#include <Types.h>
-#include <Util.h>
+#include "Communicator.h"
+#include "Logger.h"
+#include "Properties.h"
+#include "Proxy.h"
+#include "Types.h"
+#include "Util.h"
+
```

Comment:
You should try to build without the DisableWarnings, to verify we really need it.